README.md

```diff
@@ -1 +1,54 @@
-# sahemProject
\ No newline at end of file
+# sahemProject
+
+
+### About 
+sahem is a platform that links creative youth from Gaza strip who lake the suffecient fund to start their own projects with donors from all around the world.
+
+---
+
+### Problem statement:
+The existence of guys with alot of creative and development ideas who lacke the required money to fund thier projects while on the other hand, there are alot of people all around the world who are interesting in gaza economy development and are willing to support people with creative ideas.
+
+---
+
+### Our solution:
+Creating a website that grants creative people the opportionity to create and present thier ideas globally and be given the chance to be funded while at the same time allow donors who are interested in development in gaza from all around the world to view these projects, search with their criteria of interest and fund their desirable projects.
+
+---
+
+### User Story
+* As a user I want to be able to register at the website.
+* As a user I want to be able to login to my account.
+* As a register user I want to be able to create project proposal.
```

Comment:
Is there a difference between a user and a registered user?

Reply:
user is the guest and i will edit it